src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -151,6 +151,10 @@ Copyright (c) .NET Foundation. All rights reserved.
       />
   </Target>
 
+  <PropertyGroup  Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), 5.0))">
+    <_EnableDefaultWindowsPlatform Condition="'$(_EnableDefaultWindowsPlatform)' == ''" >false</_EnableDefaultWindowsPlatform>
```

Comment:
I think we also wont to set `UseOSWinMdReferences` to false here: https://github.com/microsoft/msbuild/pull/5391#discussion_r434203303